crates/ruff_python_semantic/src/model.rs

```diff
@@ -191,8 +191,9 @@ impl<'a> SemanticModel<'a> {
             .map_or(false, |binding| binding.kind.is_builtin())
     }
 
-    /// Return `true` if `member` is unbound.
-    pub fn is_unbound(&self, member: &str) -> bool {
+    /// Return `true` if `member` is an "available" symbol, i.e., a symbol that has not been bound
+    /// in the current scope, or in any containing scope.
+    pub fn is_available(&self, member: &str) -> bool {
```

Comment:
I decided to rename this. "Unbound" means something different. This method is used when autofixing, to check whether we can introduce a new symbol into a file -- so, e.g., if we're going to import `foo`, we want to make sure that `foo` wasn't defined in any scope, where it might conflict.